### ISSUE TITLE
Design inconsistency in Cyrillic

### ISSUE THREAD

**Issue by Buernia** [notofonts/latin-greek-cyrillic#446]
### Font Version
Version 2.013

### Character data
Җҗⷤ 
ҚқҜҝҟҠҡⷦ
ӅӆԒԓⷧ

### Screenshot

![图片](https://github.com/notofonts/latin-greek-cyrillic/assets/11886946/d54be2d1-5e9a-48a8-85e9-13bdd4985794)

**Comment by simoncozens** [comment]
Yes, we had the base Cyrillic upgraded earlier this year, but not the extended Cyrillic to match. It's something I'd like to get fixed.
